aws_topology/stackstate_checks/aws_topology/resources/elb_classic.py

```diff
@@ -69,6 +68,7 @@ def process_elb_page(self, elbs):
                     tags = tag_result.get("Tags", [])
             self.process_elb(LoadBalancerData(elb=data, tags=tags, instance_health=instance_health))
 
+    @set_required_access_v2("elasticloadbalancing:DescribeLoadBalancers")
```

Comment:
@Eelviny  should we put this decorator on `collect_elbs` function which is actually calling `DescribeLoadBalancers` API?

Reply:
Not here - this change is in line with all other resources. collect_elbs is a paginator function to abstract the describe_load_balancers call, so we need to place the set_required_access on the function that calls the paginator